src/test/resources/design-docs/example/views/boolean/map.js

```diff
@@ -0,0 +1,3 @@
+function(doc) {
+  emit(doc.contentArray[0].boolean, null);
+}
```

Comment:
Be sure all newly added files have trailing new lines. (Set up your editor to always add them).